src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -59,6 +60,12 @@
     public CaseDocument generate(CaseData caseData, String authorisation) {
         DocmosisTemplates templateId = TWO_V_ONE.equals(MultiPartyScenario
             .getMultiPartyScenario(caseData)) ? N181_2V1 : N181;
+
+        if (TWO_V_ONE.equals(MultiPartyScenario.getMultiPartyScenario(caseData))
```

Comment:
Please store the result of MultiPartyScenario.getMultiPartyScenario(caseData) in line 61 and reuse it.

Reply:
Done